pymc/model.py

```diff
@@ -1860,6 +1862,74 @@ def Deterministic(name, var, model=None, dims=None, auto=False):
     return var
 
 
+def _get_scaling(total_size, shape, ndim):
+    """
+    Gets scaling constant for logp.
+
+    Parameters
+    ----------
+    total_size: Optional[int|List[int]]
+        size of a fully observed data without minibatching,
+        `None` means data is fully observed
+    shape: shape
+        shape of an observed data
+    ndim: int
+        ndim hint
```

Comment:
The type hints (at least for `total_size` and `ndim` should go into the function signature, so tools like `mypy` can discover them.